src/test/kotlin/arrow/proofs/refinement/RefinementTests.kt

```diff
@@ -8,5 +8,6 @@ class RefinementTests{
     @Test
     fun `NonEmptyArray should not be empty`() {
         val x: NonEmptyArray = NonEmptyArray(emptyArray())
+        consumer.TwitterHandle("@admin")
```

Comment:
Why creating a TwitterHandle here?

Reply:
Good point! It was added by @i-walker . Please, @i-walker :pray:

Reply:
Btw tests for refined types are within `compiler-plugin`
`arrow.meta.plugins.refinement.RefinementTests`

Reply:
Right, they are here as well for the Gradle test in Arrow Meta IDE Plugin

Reply:
I removed it temporarily.. if in the end was wanted, we can create a separate test for it 👍